pkg/operator/controller/externaldns/status_test.go

```diff
@@ -557,7 +557,7 @@ func fakeExternalDNS() *operatorv1alpha1.ExternalDNS {
 				Type: operatorv1alpha1.ProviderTypeAWS,
 				AWS: &operatorv1alpha1.ExternalDNSAWSProviderOptions{
 					Credentials: operatorv1alpha1.SecretReference{
-						Name: testSecretName,
+						Name: test.OperandName,
```

Comment:
Why this change? And why it didn't impact any before existing tests?

Reply:
Sorry I missed the change in `pkg/operator/controller/externaldns/controller_test.go` where the secren name was replaced with the operand name. That answers the second question. But I'm still curious why the secret name was replaced by the operand. Maybe because of this https://github.com/openshift/external-dns-operator/pull/127#discussion_r848920447.

Reply:
Removed the change